xpdacq/tests/test_beamtimeSetup.py

```diff
@@ -94,9 +94,9 @@ def test_start_beamtime(self):
         self.assertEqual(self.experimenters, self.bt.get('bt_experimenters'))
         self.assertEqual(self.wavelength, self.bt.get('bt_wavelength'))
         self.assertEqual(os.getcwd(), self.home_dir)
-        # FIXME
-        # strtScnLst = ['bt_bt.yml','ex_l-user.yml','sa_l-user.yml','sp_ct_0.1.yml','sp_ct_0.5.yml','sp_ct_1.yml','sp_ct_5.yml','sp_ct_10.yml','sp_ct_30.yml']
-        # self.assertEqual(newobjlist,strtScnLst)
+        # test prepoluate ScanPlan
+        self.assertEqual(len(self.bt.scanplans), 3)
+        self.assertEqual(self.bt.scanplans[0]['sp_args'], (5,))
```

Comment:
I think the what we want to test here, if possible, is that the right scanplans were pre-created successfully, not that they work successfully per se.

Based on cj's `expected_sp_args` list it looks as if `ct_60` is created twice, so that test should fail....

If "min" behavior is tricky, let's leave it at `0.1` instead of `min`, but put the `min` behavior in v0.6 release as an issue.  Here we are fixing something that is not broken.....though it will be easier to fix it soonish before it is broken.

Summary:
1) create issue on v0.6 for `min` behavior
2) finalize the list of pre-created ct's
3) code them (basically done)
4) code the tests to check they were successfully created (maybe read the yml's to make sure they were written or something?)
5) pull in and close the issue.